plugin-server/src/worker/ingestion/person-state.ts

```diff
@@ -213,48 +193,49 @@ export class PersonState {
             )
         }
 
-        const update: Partial<Person> = {}
-        const updatedProperties = this.updatedPersonProperties(personFound.properties || {})
-
-        if (!equal(personFound.properties, updatedProperties)) {
-            update.properties = updatedProperties
-        }
-        if (this.updateIsIdentified && !personFound.is_identified) {
```

Comment:
Not sure why tests haven't run but you can't really remove `updateIsIdentified` and that bug should be caught by tests.